Add VMWare Tanzu notes. Also differentiate between supported and tested k8s (#719)
stackabletech/documentation: modules/ROOT/pages/kubernetes/ibm-cloud.adoc

= IBM cloud

> IBM cloud uses a non-standard Kubelet state directory.

Reviewer: ```suggestion
IBM Cloud uses a non-standard Kubelet state directory.
```

> Therefore installing secret-operator and listener-operator on IBM cloud requires a special treatment.
> Please read on the xref:secret-operator:installation.adoc#_ibm_cloud[secret-operator installation guide] and the xref:listener-operator:installation.adoc#_ibm_cloud[listener-operator installation guide] for details.

Reviewer: ```suggestion
Therefore, installing the secret-operator and listener-operator on IBM Cloud requires  special handling.
Please refer to the xref:secret-operator:installation.adoc#_ibm_cloud[secret-operator installation guide] and the xref:listener-operator:installation.adoc#_ibm_cloud[listener-operator installation guide] for details.
```
